simplegrid/world.py

```diff
@@ -117,20 +118,29 @@ def step(self):
         self.counts = Counter(creature.__class__.__name__ for creature in self.creatures.values())
 
         game_active = len(self.counts) == 2
+        if not game_active:
+            if len(self.counts) == 0:
+                print('draw')
+            else:
+                print(list(self.counts.elements())[0], 'wins')
```

Comment:
Ik zou `.most_common(...)` gebruiken. Hoef je geen list cast, plus dat werkt ook als we later meerdere finalisten hebben.

Reply:
Ok. Had ik ook al naar gekeken maar dan krijg je self.counts.most_common()[0][0] en die [0][0]  vond ik niet zo mooi.